lib/tasks/export_tasks.rake

```diff
@@ -0,0 +1,241 @@
+#
+# export_tasks.rake is a debugging tool to extract tasks from the
+# current foreman instance.
+# 
+# Run "foreman-rake export_tasks" to export tasks whcih are not listed
+# as successful. 
+# To export all tasks "foreman-rake export_tasks tasks=all"
+
+desc 'Export Dynflow Tasks'
+  task :export_tasks => :environment do
```

Comment:
Yes please: the directory under foreman_tasks/tasks would be better fit.
Yes: the standard is 2, but there are still too many 4-line ones, starting here https://github.com/theforeman/foreman-tasks/pull/106/files#diff-10af87cb135065f8f7ed35b14a284ccbR12